### PR TITLE
fix: roles should be a alias

### PR DESCRIPTION
**Pull request by echarles**

### CODE REVIEW

File: docs/docs/usage/index.mdx [hunk 1]
```diff
@@ -39,6 +39,45 @@ clouder ssh-key ls
...
+clouder k8s create $DATALAYER_CLUSTER_NAME
+clouder k8s kubeconfig $DATALAYER_CLUSTER_NAME
+clouder k8s
+clouder k8s use $DATALAYER_CLUSTER_NAME
+clouder k8s
+clouder info
+```
+
+```bash
+clouder k8s create-nodepool $DATALAYER_CLUSTER_NAME router --flavor b2-15 --min 0 --desired 1 --max 10 --role router --roles router --xpu cpu
```
Comment (fcollonval) [inline docs/docs/usage/index.mdx:62]: is `--roles <name>` ok or is the `=` sign mandatory?

Comment (echarles) [reply docs/docs/usage/index.mdx:62]: trying now, indeed, it is not clear...

Comment (echarles) [reply docs/docs/usage/index.mdx:62]: I confirm it is working without the `=` , so the PR is good to me.

Comment (fcollonval) [reply docs/docs/usage/index.mdx:62]: Thanks for quickly testing

**Review by fcollonval — APPROVED** [review]
Thanks @echarles

**Pull request merged**